### DISCUSSION TITLE
Chat comms channels

### DISCUSSION THREAD

**Discussion by bmann** [IPFSConnect/community#2] [Istanbul 2023]
We think we need a chat interface for at least the day of the conference, especially for in person workshop support.

What else is being used across Labweek / Devconnect generally?

**Comment by bmann** [comment]
We are experimenting with a category and set of channels in the existing IPFS Discord.

We do think we want to have a bridge into Matrix and maybe also into Telegram?

**Comment by BigLep** [comment]
Bridging could have improved but here is a dump of UX caveats with Discord <> Matrix bridging that was done in late 2022: https://pl-strflt.notion.site/IPFS-Matrix-Homeserver-Documentation-31481e76843547b7ab5160a87eed2b9f#d086b8c2851040e0b7bc5e8c1f444f2a
